Neuron/Sections/Dapp/DAppCommand.swift

```diff
@@ -61,6 +61,43 @@ struct ETHObject: Decodable {
     var to: String?
     var gasLimit: Double?
     var gasPrice: Double?
+
+    enum CodingKeys: String, CodingKey {
+        case chainId
+        case value
+        case data
+        case from
+        case to
+        case gasLimit
+        case gasPrice
+    }
+
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
+        if let chainId = try? values.decode(Int.self, forKey: .chainId) {
```

Comment:
Does `self.chainId = try? values.decode(Int.self, forKey: .chainId)` work? If so that's simpler.